README.md

```diff
@@ -1,148 +1,111 @@
+        "get": {
+            "source": "./test/etc/barGet.js"
         },
-        "accountInfo/old": {
-            "get": {
-                "source": "./routes/getAccountInfoOld.js"
-            }
+        "post": {
+            "source": "./test/etc/barPost.js"
         }
     }
 }
 ```
-The second level keys "acceptcookies", "accountInfo", "cancelPlan", "accountInfo/old" is url path.
-The third level keys are HTTP method names.
-The value of "source" is the route handler script's relative path from app root.
-
+This declares the route name, http method, and handler file on disk. this
+module will install these routes onto a restify server for you. The
+corresponding handler file would look like:
 
-## Usage Example
 ```javascript
- enroute.createRoute({
-            //restify server object
-            server: server,
-            //bunyan log object
-            log: log,
-            //array of route config object, the object includes route configuration file path
-            //and an optional baseUrl String, it will be prepend
-            //for every route path specified in route configuration file. user can also provide
-            //route definition object directly by doing
-            //routeConf: [{routeDefinitionObj : routeObj}],
-            routeConf: [{filePath :  routeScriptRootPath + '/route.json'}],
-            //user can specify an array of universal middleware function before all routes
-            preMiddleware : preHandlerArr,
-            //user can specify an array of universal middleware function after all routes
-            postMiddleware : postMiddleware,
-            //the root path of all route scripts, it can be different from route.json path
-            scriptPath : routeScriptRootPath
-        }, function() {
-        //user can specify optional callback function
-            client.get('/hello',
-                function(err, req, res, obj) {
-                if (err) {
-                    done(err);
-                }
-                assert.equal(obj.data, 'Hello world!',
-                          'Hello World endpoint response.');
-                done();
-            });
-        });
-    });
+module.exports = function handler(req, res, next) {
+    res.send(200, 'Hello World');
+    next()
+};
 ```
-Routes can be injected into Restify server either before or after server start.
-
-
-## Route Handler Script Example
-```javascript
-'use strict';
-// Node core modules
-var url = require('url');
 
+## API
+Synopsis: `enroute(opts, cb)`
 
-// Third party modules specified in package.json
-var _ = require('lodash');
-var falcor = require('falcor');
-var verror = require('verror');
+Installs routes as defined in opts into a restify server, invokes the callback
+when done.
+* `opts`: The options object containing
+    * `opts.server` The restify server to install the routes on to.
+    * `opts.config` The POJO of the enroute config.
+    * `opts.configPath` The path to the enroute config on disk.
```

Comment:
Yep, they are - would recommend documenting that here (and also add [] to indicate optionalness as you have in the JSDocs).